draft-linus-trans-gossip-ct.md

```diff
@@ -166,9 +166,11 @@ SCTs and certificate chains.
 
 HTTPS servers store SCTs and certificate chains received from clients
 and later share them with CT auditors by either posting them or making
-them available on a .well-known URL. This is described in
+them available on a .well-known URL. [Making them available seems like it would not scale well, though I do like the idea for crawlers to harvest them] This is described in
```

Comment:
Well, the idea was we limit the store in several ways so the space required is minimal.  Figure a webserver knows about 10 logs, they host 10 vhosts, have 15 certificates in play: 150 possible SCTs plus ~450 certificates.  ~10 MB of space?  (They would not store SCTs/certs for sites they are not responsible for.)